src/generic/clipboard/paste-button/index.jsx

```diff
@@ -2,10 +2,12 @@ import { useRef, useState } from 'react';
 import PropTypes from 'prop-types';
 import { OverlayTrigger, Popover } from '@openedx/paragon';
 
-import { PopoverContent, PasteComponentButton, WhatsInClipboard } from './components';
+import { PopoverContent, PasteButtonComponent, WhatsInClipboard } from './components';
 import { clipboardPropsTypes, OVERLAY_TRIGGERS } from './constants';
 
-const PasteComponent = ({ handleCreateNewCourseXBlock, clipboardData }) => {
+const PasteButton = ({
```

Comment:
Maybe this we can call as `ClipboardPasteComponent` and `inside we will have `ClipboardPasteButton`?

Component - because it consist of other parts like button in this case and etc.
Button - single element that will a part of component

Reply:
Renamed